workflows/inngest/src/index.ts

```diff
@@ -295,8 +295,7 @@ export class InngestRun<
           steps,
           stepResults: snapshot?.context as any,
           resumePayload: resumeDataToUse,
-          // @ts-ignore
-          resumePath: snapshot?.suspendedPaths?.[steps?.[0]] as any,
+          resumePath: snapshot?.suspendedPaths?.[steps?.[0] ?? ''] as any,
```

Comment:
We should do something along these lines
```suggestion
          resumePath: steps?.[0]  ? snapshot?.suspendedPaths?.[steps?.[0]] as any : undefined,
```